src/modules/categories/components/CategoryListItem.js

```diff
@@ -78,13 +85,33 @@ class CategoryListItem extends React.Component<PropsType> {
     )
   }
 
+  getMatchedContent (numWordsSurrounding: number): ?Highlighter {
+    const { query, theme, category } = this.props
+    const textToHighlight = this.contentMatcher.getMatchedContent(query, category.contentWithoutHtml, numWordsSurrounding)
```

Comment:
Any reason why you have to initialize contentMatcher for the whole component?

Reply:
The main reason why I did it, is because that is what I did in the webapp and nobody complained. Furthermore it kind of feels incorrect to reinstantiate the contentMatcher on every render